content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,93 @@
+
+## What is Notary Project and Notation
+
+As containers and cloud native artifacts become common deployment units, users want to be sure that the artifacts in their environments are authentic and unmodified. The Notary Project is a set of specifications and tools intended to provide cross-industry standards for securing software supply chains through signing and verification, signature portability, and key/certificate management. 
+
+Notation is a sub-project of the Notary Project GitHub organization. It has `notation` CLI and two Golang libraries, which are implemented from the latest [Notary Project specifications](https://github.com/notaryproject/specifications/releases/tag/v1.0.0). Notation was started in [Dec 2019](https://github.com/notaryproject/meeting-notes/blob/main/meeting-notes-2019.md#notary-v2-kickoff-meeting) and the code  has matured through a series of minor and RC releases over the last few years, the first release [v0.7.0-alpha.1](https://notaryproject.dev/blog/2021/announcing-notation-alpha1/) was available in Oct 2021 and reached [v1.0.0-RC.7](https://notaryproject.dev/blog/2023/announcing-notation-rc6/) in May 2023.
+
+To learn more about the overall Notary Project and terminologies, see [Notary Project Overview](https://github.com/notaryproject/.github#notary-project-overview) and [FAQ - Notary Project Terms](https://notaryproject.dev/docs/faq/#notary-project-terms).
+
+## Highlights
+
+There are some remarkable new features and updates in the first major release. This blog post walks through the major updates in this release.
+
+### Specifications
+
+[Notary Project specifications](https://github.com/notaryproject/specifications/releases/tag/v1.0.0) reach its first major release. All specifications, requirements, scenarios, threat model, and security audit reports are available in this release. To interoperate with Notary Project tooling or develop your own reference implementation, follow the latest Notary Project specifications for details.
+
+### Strengthen signing and verification functionalities
+
+From the software producer's perspective, signing a software artifact is the way to ensure authenticity and trust when distributing a software to consumers. Notary Project provides the follow core capabilities for the signing experience.
+
+- Sign artifacts using signing keys stored securely in key management system (KMS) using vendor or customer specific plugins
+- Store signatures using [OCI Image Manifest](https://github.com/opencontainers/image-spec/blob/v1.1.0-rc3/spec.md) for standardization and portability
```

Comment:
Same comment here as we made on Shiwei's blog. Mention  [image-spec v1.0.2](https://github.com/opencontainers/image-spec/tree/v1.0.2)
 as well